tests/integration/advanced/graph/fluent/test_graph.py

```diff
@@ -120,10 +118,10 @@ def _send_batch_and_read_results(self, schema, graphson, add_all=False, use_sche
         ep = self.get_execution_profile(graphson)
         batch = DseGraph.batch(session=self.session,
                                execution_profile=self.get_execution_profile(graphson, traversal=True))
-        for data in six.itervalues(datatypes):
+        for data in datatypes.values():
             typ, value, deserializer = data
             vertex_label = VertexLabel([typ])
-            property_name = next(six.iterkeys(vertex_label.non_pk_properties))
+            property_name = next(vertex_label.non_pk_properties.keys())
```

Comment:
keys() on a dict returns a dictionary view object rather than a sequence so this needs to be explicitly converted to an iterator.  So this should instead be:

```
property_name = next(iter(vertex_label.non_pk_properties.keys()))
```